docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -0,0 +1,289 @@
+
+.. contents:: :local:
+
+Step 1: Connecting your Azure account
+-----------------------------------
+
+Hopsworks.ai deploys Hopsworks clusters to your Azure account. To enable this, you have to
+create a service principal and a custom role for Hopsworks.ai granting access
+to either a subscription or resource group.
+
+Step 1.1: Creating a service principal for Hopsworks.ai
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On Hopsworks.ai, go to *Settings/Cloud Accounts* and choose to *Configure* Azure:
+
+.. _connect-azure-0.png: ../../../_images/connect-azure-0.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-0.png
+    :alt: Cloud account settings
+    :target: `connect-azure-0.png`_
+    :align: center
+    :figclass: align-center
+
+Select *Add subscription key*:
+
+.. _connect-azure-0.1.png: ../../../_images/connect-azure-0.1.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-0.1.png
+    :alt: Add subscription keys
+    :target: `connect-azure-0.1.png`_
+    :align: center
+    :figclass: align-center
+
+The Azure account configuration will show you the required steps and permissions.
+Ensure that you have the Azure CLI installed (`Install the Azure CLI <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli>`_)
+and are logged in (`Sign in with Azure CLI <https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli>`_).
+
+Copy the Azure CLI command to add a service principal for Hopsworks.ai from the first step and open a terminal:
+
+.. _connect-azure-1.png: ../../../_images/connect-azure-1.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-1.png
+    :alt: Connect your Azure Account
+    :target: `connect-azure-1.png`_
+    :align: center
+    :figclass: align-center
+
+Paste the command into the terminal and execute it:
+
+.. _connect-azure-2.png: ../../../_images/connect-azure-2.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-2.png
+    :alt: Add service principal
+    :target: `connect-azure-2.png`_
+    :align: center
+    :figclass: align-center
+
+At this point, you might get the following error message.
+This means that your Azure user does not have sufficient permissions to add the service principal.
+In this case, please ask your Azure administrator to add it for you or give you the required permissions.
+
+.. code-block:: bash
+
+    $ az ad sp create --id d4abcc44-2c40-40bd-9bba-986df591c28f
+    When using this permission, the backing application of the service principal being created must in the local tenant
+
+Step 1.2: Creating a custom role for Hopsworks.ai
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Proceed to the Azure Portal and open either a *Subscription* or *Resource Group* that you want to use for Hopsworks.ai.
+Select *Add* and choose *Add custom role*. Granting access to a *Subscription* will grant access to all *Resource Groups*
+in that *Subscription*. If you are uncertain if you want that then start with a *Resource Group*.
```

Comment:
If you are uncertain that it is what you want, then start with a *Resource Group*.

Reply:
done